### DISCUSSION TITLE
The template image is displayed incorrectly

### DISCUSSION THREAD

**Discussion by lipan1024** [Budibase/budibase#3631] [Help] [answered]
![image](https://user-images.githubusercontent.com/86212306/144344358-b322f432-ebce-476d-9c10-cc3d34bf553d.png)  
I don't know what the problem is...

**Comment by mjashanks** [comment, marked as answer]
Hmm - I just used this template, and the images are working ok for me.

If you open up your developer tools & network tab, do you see any failed requests (404?) for these images? Are you able to post the response body?

Also - are you using cloud or self-host?

**Reply by lipan1024** [reply]
Thank you for your reply. I think it's probably the Internet problem
